radiant/experiments/sql/create_iceberg_catalog.sql

```diff
@@ -0,0 +1,14 @@
+CREATE EXTERNAL CATALOG 'iceberg'
+COMMENT "External catalog to Apache Iceberg on MinIO"
+PROPERTIES
+(
+  "type"="iceberg",
+  "iceberg.catalog.type"="rest",
+  "iceberg.catalog.uri"="http://iceberg-rest:8181",
+  "iceberg.catalog.warehouse"="warehouse",
+  "aws.s3.access_key"="admin",
+  "aws.s3.secret_key"="password",
+  "aws.s3.endpoint"="http://minio:9000",
+  "aws.s3.enable_path_style_access"="true",
+  "client.factory"="com.starrocks.connector.share.iceberg.IcebergAwsClientFactory"
```

Comment:
This was changed to add `share` in the path. Otherwise, StarRocks wasn't able to initialize the `S3` client 

ref.: https://github.com/StarRocks/starrocks/issues/51660#issuecomment-2475840093